modules/ROOT/pages/schema/constraints/list-constraints.adoc

```diff
@@ -130,13 +134,13 @@ SHOW CONSTRAINTS YIELD *
 | 12 | "movie_tagline"         | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["tagline"]                  | NULL          | "independent"  | NULL                    | "STRING | LIST<STRING NOT NULL>" | NULL              | "CREATE CONSTRAINT `movie_tagline` FOR (n:`Movie`) REQUIRE (n.`tagline`) IS :: STRING | LIST<STRING NOT NULL>"     |
 | 33 | "movie_title"           | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["title"]                    | NULL          | "independent"  | NULL                    | "STRING"                         | NULL              | "CREATE CONSTRAINT `movie_title` FOR (n:`Movie`) REQUIRE (n.`title`) IS :: STRING"                                 |
 | 17 | "node_uniqueness_param" | "NODE_PROPERTY_UNIQUENESS"         | "NODE"         | ["Book"]       | ["prop1"]                    | NULL          | "undesignated" | "node_uniqueness_param" | NULL                             | {indexConfig: {}} | "CREATE CONSTRAINT `node_uniqueness_param` FOR (n:`Book`) REQUIRE (n.`prop1`) IS UNIQUE"                           |
-| 2  | "node_vector_constraint" | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["embedding"]                | NULL         | ""             | NULL                    | "VECTOR<INTEGER32 NOT NULL>(42)" | NULL              | "CREATE CONSTRAINT `node_vector_constraint` FOR (n:`Movie`) REQUIRE (n.`embedding`) IS :: VECTOR<INTEGER32 NOT NULL>(42)"         |
+| 2  | "node_vector_constraint" | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["embedding"]                | NULL         | "independent"             | NULL                    | "VECTOR<INTEGER32 NOT NULL>(42)" | NULL              | "CREATE CONSTRAINT `node_vector_constraint` FOR (n:`Movie`) REQUIRE (n.`embedding`) IS :: VECTOR<INTEGER32 NOT NULL>(42)"         |
```

Comment:
attempting to fix the misalignment
<img width="1503" height="772" alt="image" src="https://github.com/user-attachments/assets/ee00de24-d952-450b-a606-6ff40b532b51" />
```suggestion
| 2  | "node_vector_constraint" | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["embedding"]                | NULL         | "independent"  | NULL                    | "VECTOR<INTEGER32 NOT NULL>(42)" | NULL              | "CREATE CONSTRAINT `node_vector_constraint` FOR (n:`Movie`) REQUIRE (n.`embedding`) IS :: VECTOR<INTEGER32 NOT NULL>(42)"         |
```